### PR TITLE
Add geometry creation from the relation editor widget

### LINKED ISSUE #4555: Create child feature with geometry from the relation editor (#32528)
Original commit: https://github.com/qgis/QGIS/commit/ba925bb36d6399bfc29c36dde4070b190cc5a789 by mhugo

* [FEATURE][needs-docs] Create geometric child feature from relation editor

### PR DESCRIPTION
**Pull request by troopa81**
Fixes #4555 

Goal:
This PR add some documentation for the following [developpement](https://github.com/qgis/QGIS/pull/32528)

Ticket(s): #

- [ ] Backport to LTR documentation is required

No need to backport to LTR, this feature is only in 3.12

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 1]
```diff
@@ -734,6 +734,9 @@ a table. And there are also some buttons available. Let's review them shortly:
 * The |saveEdits| button is for saving all the edits.
 * The |newTableRow| button will add a new record to the airport layer attribute table.
   And it will assign the new airport to the current region by default.
+* The |newTableRowWithGeom| is the same as |newTableRow| but let you digitize the
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: Why not 
```suggestion
* The |capturePoint| is the same as |newTableRow| but lets you digitize the
```
given that the substitution already exists? I'm afraid that this one gets cleared if someone runs the [find_set subst](https://github.com/qgis/QGIS-Documentation/blob/master/scripts/find_set_subst.py) script

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 2]
```diff
@@ -734,6 +734,9 @@ a table. And there are also some buttons available. Let's review them shortly:
...
+  airport geometry in the map canvas beforehand. Note that icon will change according
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]:
```suggestion
  airport geometry in the map canvas beforehand. Note that the icon will change according
```

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 3]
```diff
@@ -1019,6 +1022,8 @@ table.
    :width: 1.5em
 .. |newTableRow| image:: /static/common/mActionNewTableRow.png
    :width: 1.5em
+.. |newTableRowWithGeom| image:: /static/common/mActionCapturePoint.png
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]:
```suggestion
.. |capturePoint| image:: /static/common/mActionCapturePoint.png
```

**Comment by troopa81** [comment]
@DelazJ I accepted your suggestions, do you want me to squash the commits or is it OK this way?

**Comment by DelazJ** [comment]
Thanks @troopa81. Squashed while merging.

**Comment by troopa81** [comment]
Thanks for reviewing @DelazJ

**Pull request merged**